app/indices/account_index.rb

```diff
@@ -12,7 +12,7 @@
   indexes :id,                                 as: :account_id
   indexes users.id,                            as: :user_id
 
-  set_property field_weights: { name: 2 }
+  set_property field_weights: { name: 2 }, group_concat_max_len: 32_000
```

Comment:
it should be `32.kylobites - 1` Just to make it the same as for Oracle at least.